src/components/MenuBar/MenuBar.tsx

```diff
@@ -14,43 +14,27 @@ import {
   DefaultButton as Button,
   DefaultPopover as Popover,
   DefaultLink as Link,
-  DefaultDivider as Divider
+  DefaultDivider as Divider,
+  DefaultSkeleton as Skeleton
 } from '@common/components'
 import SettingsIcon from '@mui/icons-material/Settings'
 import HelpIcon from '@mui/icons-material/Help'
 import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
 import { useTranslation } from 'react-i18next'
+import { ListItemIcon } from '@mui/material'
+import { Logout } from '@mui/icons-material'
+import { AuthContext, Topic, LearningPath } from '@services'
```

Comment:
Nur damit ich das nicht vergesse. Haben bisher noch keinen Ordner daür aber bietet sich an.
```suggestion
// TODO: Move it into @common/hooks since it is reused in LocalNav
```